torch/_dynamo/variables/user_defined.py

```diff
@@ -692,6 +699,24 @@ def const_getattr(self, tx: "InstructionTranslator", name):
         return super().const_getattr(tx, name)
 
 
+class UserDefinedExceptionClassVariable(UserDefinedClassVariable):
+    def __init__(self, value, **kwargs):
+        super().__init__(value, **kwargs)
+        self.exc_vt = variables.ExceptionVariable(value, ())
```

Comment:
Why does this own an ExceptionVariable? If we need it (in getattr), then it might be better to refactor ExceptionVariable's `__getattr__` so that it can be reused?

It sounds like what we're trying to model is: the Exception class has some class attributes and those should be accessible from UserDefinedExceptionClassVariable and ExceptionVariable

Reply:
It shouldn't own an exception variable in this case. I'll remove it.